docs/PROJECT_SETUP.md

```diff
@@ -45,6 +46,49 @@ Note that `Datadog::Statsd` is not _required_.  Adding it, however, will enable
 #### ★ Best Practice ★
 Keep the global namespace short.  For example, memcached has a limit of 250 characters for key length.
 
+#### More Complex Rails Configuration
+
+In any real-world project, the need to run multiple caching strategies or setups is likely to arise. In those cases, it's often advantageous
+to keep a DRY setup, with multiple caching clients sharing the same config. Because Rails initializers run after the environment-specific
+config files, a sane way to manage this is to keep client network settings int he config files, then reference them from the initializer.
```

Comment:
👍 Thanks for including this! If others use this gem, I'd anticipate this being really helpful.

Reply:
Oh and `# config/environments/staging` on #L64 -> `production`